tools/gyp/pylib/gyp/input.py

```diff
@@ -709,6 +698,9 @@ def FixupPlatformCommand(cmd):
 
 def ExpandVariables(input, phase, variables, build_file):
   # Look for the pattern that gets expanded into variables
+  def to_utf8(s):
+    return s if isinstance(s, str) else s.decode("utf-8")
```

Comment:
Style nit: single quotes.

Reply:
The Python Software Foundation is [going in the opposite direction](https://github.com/psf/black#strings) so the muscle memory is building up but will fix.